build.sbt

```diff
@@ -326,13 +326,13 @@ lazy val sshd = project
       name := "ammonite-sshd",
       libraryDependencies ++= Seq(
         // sshd-core 1.3.0 requires java8
-        "org.apache.sshd" % "sshd-core" % "1.2.0",
-        "org.bouncycastle" % "bcprov-jdk15on" % "1.56",
+        "org.apache.sshd" % "sshd-core" % "1.6.0",
```

Comment:
Java 6 EOL more than 4 years (https://developers.slashdot.org/story/13/06/20/1819245/java-6-eold-by-oracle).
May be it's time to drop support of Java 6?